quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Build.java

```diff
@@ -111,8 +110,13 @@ public List<OptionCategory> getOptionCategories() {
     }
 
     private void exitWithErrorIfDevProfileIsSetAndNotStartDev() {
```

Comment:
```suggestion
    private void exitWithErrorIfDevProfileIsSet() {
```
nitpick: not related to start-dev anymore